clients/js/helpers.go

```diff
@@ -0,0 +1,64 @@
+package jsclient
+
+import (
+	"bytes"
+	"regexp"
+)
+
+func camelCase(src string, capFirst bool) string {
```

Comment:
worth pulling this out into a more general helper file. I use something much primitive in another part of the code: https://github.com/Clever/wag/blob/master/swagger/parameter.go#L314